pypesto/objective/base.py

```diff
@@ -326,13 +326,55 @@ def update_from_problem(
 
         self.pre_post_processor = pre_post_processor
 
+    def check_grad_multi_eps(
+            self,
+            *args,
+            multi_eps: AbstractSet = None,
```

Comment:
Motivation for using `AbstractSet` here? What about `Iterable` ? Should be all we need and more specific whats actually required.

Reply:
It seemed appropriate for an unordered collection of unique numbers. The method would still work with `Iterable`, which is probably more familiar to most users, so I'll switch to that, thanks.